sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/VmMetadataResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal class VmMetadataResponse
```

Comment:
looks like this is a short-lived object.
Should this be a `struct`?

Reply:
Discussed offline - This is not on hot path and is used only 24hrs duration for serialization.